public/templates/templates.go

```diff
@@ -181,3 +362,94 @@ func (t *Templater) RenderNoNavsTemplate(w io.Writer, data interface{}, mainTmpl
 func (t *Templater) RenderEmail(emailTemplate Template) *template.Template {
 	return template.Must(template.New("forgotEmail.tmpl").ParseFS(tmpls, emailTemplate.GetString()))
 }
+
+func GetValidPermissions(p permissions.Permissions) (m map[string]bool) {
```

Comment:
I'm a bit confused by this function - what does it return and why is it necessary?

Reply:
It is the logic for valid permissions and parent permission, this may change later but for the moment it works for validating permissions